canonical-webteam/global-nav
# Make login menu optional

Make it so `logins` and `products` can be set to `false` when calling `createNav`.

Fixes https://github.com/canonical-webteam/global-nav/issues/59

Bump version to `v1.2.0` for this change.

QA
--

``` bash
yarn install
yarn build
chromium-browser index.html  # Or otherwise open index.html in your browser
```

Check the nav works. Now, in index.html, try changing line 56 to:

``` html
<script>canonicalGlobalNav.createNav({showLogins: false});</script>
```

Refresh `index.html` in your browser and check the nav behaves as expected.
